tools/topology/sof-cml-demux-rt5682.m4

```diff
@@ -0,0 +1,178 @@
+#
+# Topology for Cometlake with rt5682 codec.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Cannonlake DSP configuration
+include(`platform/intel/cnl.m4')
+include(`platform/intel/dmic.m4')
+
+DEBUG_START
+
+undefine(`SSP_INDEX')
+define(`SSP_INDEX', ifelse(PLATFORM, `whl', `1',
+	ifelse(PLATFORM, `cml', `0', `')))
+undefine(`SSP_NAME')
+define(`SSP_NAME', ifelse(PLATFORM, `whl', `SSP1-Codec',
+	ifelse(PLATFORM, `cml', `SSP0-Codec', `')))
+
+#
+# Define the pipelines
+#
+# PCM0 <---> volume <----> SSP(SSP_INDEX)
+# PCM1 ----> volume -----> DMIC01 (dmic0 capture)
```

Comment:
@juimonen can you please also adjust the pipeline info here?

Reply:
@ranj063 yep, I'll have to go through all the files, there's a lot of copy paste... I was trying to get something fast for others to test